src/App.jsx

```diff
@@ -14,33 +17,100 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button type="button" className="button is-info is-light">
-        Sort alphabetically
-      </button>
+const SORT_BY_NAME = 'name';
+const SORT_BY_LENGTH = 'length';
+const REVERSE = 'reverse';
+let isSortParametersNull;
 
-      <button type="button" className="button is-success is-light">
-        Sort by length
-      </button>
+export function App() {
+  const goodsCopy = [...goodsFromServer];
+  const [sortType, setSortType] = useState(null);
+  const [reverseOfArray, setReverseOfArray] = useState(null);
```

Comment:
```suggestion
  const [isReversedGoods, setIsReversedGoods] = useState(false);
```
It could be just boolean so you can toggle it easily